src/ontology/mondo-edit.obo

```diff
@@ -462006,7 +462009,7 @@ is_a: MONDO:0015977 {source="MESH:C538056", source="MONDO:Redundant", source="OM
 intersection_of: MONDO:0011096 {source="OMIM:601495"} ! autosomal agammaglobulinemia
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 {source="OMIM:601495"} ! IGHM
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0020729 {source="MONDO:CLINGEN"}
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 {source="MONDO:mim2gene_medgen", source="OMIM:601495"} ! IGHM
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 {source="MONDO:mim2gene_medgen"} ! IGHM
```

Comment:
this is still on the website

Reply:
this should have been maintained `relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 {source="MONDO:mim2gene_medgen", source="OMIM:601495"} ! IGHM`

it's in the omim.owl file, but i don't see it in the mondo-omim-genes.robot.tsv file